pkg/monitor/backenddisruption/disruption_backend_sampler.go

```diff
@@ -325,6 +336,9 @@ func (b *BackendSampler) checkConnection(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	if b.auditID != "" {
+		req.Header.Set(audit.HeaderAuditID, b.auditID)
```

Comment:
audit ID needs to be unique for every request.  You probably want to use a UUID.

Reply:
Lower down, please store the audit IDs of failed requests and serialize the list of failed audit IDs in the disruption json file.

Reply:
I think including failed audit IDs in the disruption file needs a much more involved refactor and will have to wait, there's no place to put them as we're basing the whole system off intervals and using the same error message to build start/end times over multiple requests. 

For now I'm thinking just log the failed UUIDs and we'll have to correlate manually when debugging.